Tools/WinMLRunner/OutputHelper.h

```diff
@@ -273,6 +283,49 @@ class OutputHelper
         return false;
     }
 
+    void SaveEvalTimes(Profiler<WINML_MODEL_TEST_PERF> &profiler, uint32_t IterNum)
+    {
+        m_CPUWorkingDiff[IterNum] = profiler[EVAL_MODEL].GetCpuWorkingDiff();
```

Comment:
If we evaluate twice, once on the CPU and once on the GPU, m_CPUWorkingDiff, m_CPUWorkingStart, m_GPUSharedDiff, m_GPUSharedStart, and m_GPUDedicatedDiff vectors are all of size 0. There is an exception that is thrown because we are trying to write to it here.